.gitignore

```diff
@@ -1,2 +1,5 @@
 _kubevirtci/
 build/
+.idea/
+bin/
+cover.out
```

Comment:
should be part of #26 imo as it present the file

does it also present bin/ ? if so same deal